tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-product-mapper-test.php

```
@@ -1168,4 +1168,70 @@ public function test_should_sync_product_new_filter_overrides_deprecated() {
+		add_filter( 'wc_stripe_agentic_commerce_disable_checkout', $callback );
+
+		try {
+			$mapper = new \WC_Stripe_Agentic_Commerce_Product_Mapper();
+			$result = $mapper->map_product( $product );
+
+			$this->assertSame( 'true', $result['disable_checkout'] );
+		} finally {
+			remove_filter( 'wc_stripe_agentic_commerce_disable_checkout', $callback );
+			$product->delete( true );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_disable_checkout_filter_overrides_store_wide_default is not named in camelCase. (undefined)

(CamelCaseMethodName)